### PR TITLE
Docs website

### PR DESCRIPTION
**Pull request by bgschiller**

### CODE REVIEW

**Comment by bgschiller** [comment]
I think this is ready to go. In order to enable github pages hosting, an admin for this repo will have to go to settings and enable Github Pages using the `gh-pages` branch as the source. [instructions](https://help.github.com/en/articles/configuring-a-publishing-source-for-github-pages#enabling-github-pages-to-publish-your-site-from-master-or-gh-pages)

Also, we don't have anything set up to auto-generate these docs upon a new commit to master. I think Tim intends to use jenkins. I've set up `deploy_docs.sh` to do the necessary steps.

**Comment by macumber** [comment]
Looks like gh-pages is already set up?  Maybe it just does that if you create the branch?

**Comment by macumber** [comment]
Hmm the link does seem to be dead http://urbanopt.github.io/urbanopt-geojson-gem

File: .rdoc_options [hunk 1]
```diff
@@ -0,0 +1,50 @@
+--- !ruby/object:RDoc::Options
+encoding: UTF-8
+static_path: []
+rdoc_include:
+- lib
+- "/Users/brian/code/urbanopt-geojson-gem/lib"
```
Comment (macumber) [inline .rdoc_options]: @bgschiller this line should go away right?

File: .rdoc_options [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+locale:
+locale_dir: locale
+locale_name:
+main_page: RDOC_MAIN.md
+markup: markdown
+output_decoration: true
+op_dir: ./docs/.vuepress/public/rdoc
+show_hash: false
+tab_width: 8
+template_stylesheets: []
```
Comment (macumber) [inline .rdoc_options]: @bgschiller i added one last task to try to coordinate the themes between the main page, schemas page, and rdoc.  Bootstrap themes would be great if that is possible.

File: RDOC_MAIN.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+# UrbanOpt GeoJSON Gem
+
+[back to main docs](../../)
```
Comment (macumber) [inline RDOC_MAIN.md]: @bgschiller can we make this link more prominent, make sure it looks like nav?

File: docs/README.md [hunk 4]
```diff
@@ -0,0 +1,5 @@
+# UrbanOpt GeoJSON Gem
+
+<StaticLink href="/rdoc/">Rdocs</StaticLink>
```
Comment (macumber) [inline docs/README.md]: @bgschiller can we make this link more prominent, make sure it looks like nav?

**Pull request merged**